spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -8,6 +8,7 @@ module CCMS # rubocop:disable Metrics/ModuleLength
         create :legal_aid_application,
                :with_proceeding_types,
                :with_everything,
+               :with_applicant_and_address,
                populate_vehicle: true,
```

Comment:
I don't see any tests for the new attributes you've added.

Reply:
Sorry i added review tag too early